spec/API_specification/data_types.rst

```diff
@@ -0,0 +1,163 @@
+.. _data-types:
+
+Data Types
+==========
+
+    Array API specification for supported data types.
+
+A conforming implementation of the array API standard must provide and support the following data types.
+
+bool
+----
+
+Boolean (``True`` or ``False``).
+
+int8
+----
+
+An 8-bit signed integer whose values exist on the interval ``[-128, +127]``.
+
+int16
+-----
+
+A 16-bit signed integer whose values exist on the interval ``[−32,767, +32,767]``.
+
+int32
+-----
+
+A 32-bit signed integer whose values exist on the interval ``[−2,147,483,647, +2,147,483,647]``.
+
+int64
+-----
+
+A 64-bit signed integer whose values exist on the interval ``[−9,223,372,036,854,775,807, +9,223,372,036,854,775,807]``.
+
+uint8
+-----
+
+An 8-bit unsigned integer whose values exist on the interval ``[0, +255]``.
+
+uint16
+------
+
+A 16-bit unsigned integer whose values exist on the interval ``[0, +65,535]``.
+
+uint32
+------
+
+A 32-bit unsigned integer whose values exist on the interval ``[0, +4,294,967,295]``.
+
+uint64
+------
+
+A 64-bit unsigned integer whose values exist on the interval ``[0, +18,446,744,073,709,551,615]``.
+
+float32
+-------
+
+IEEE 754 single-precision (32-bit) binary floating-point number (see IEEE 754-2019).
+
+float64
+-------
+
+IEEE 754 double-precision (64-bit) binary floating-point number (see IEEE 754-2019).
+
+.. note::
+   IEEE 754-2019 requires support for subnormal (a.k.a., denormal) numbers, which are useful for supporting gradual underflow. However, hardware support for subnormal numbers is not universal, and many platforms (e.g., accelerators) and compilers support toggling denormals-are-zero (DAZ) and/or flush-to-zero (FTZ) behavior to increase performance and to guard against timing attacks.
+
+   Accordingly, subnormal behavior is left unspecified and, thus, implementation-defined. Conforming implementations may vary in their support for subnormal numbers.
+
+.. admonition:: Future extension
```

Comment:
This is not using the green box like it did in the Markdown version.

Reply:
I'm not sure why I'm unable to get the green box in rst, I tried everything in [this](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Admonitions.html) documentation.